client-js/app/elements/labrad-plot.ts

```diff
@@ -1,9 +1,17 @@
 import 'd3';
-
+import * as three from 'three';
 import {viridisData} from '../scripts/colormaps';
 import * as datavault from "../scripts/datavault";
 
+/**
+ * Import the THREE namespace from the module.
+ */
+const THREE = three.default;
```

Comment:
This throws a compiler error, but was the only way I could get the library to play nicely with TypeScript. If someone can work out how to import the library without this hack, please let me know, I wasted a few hours trying and ultimately moved on.